example/src/services/SetupService.ts

```diff
@@ -9,6 +9,9 @@ export const SetupService = async (): Promise<boolean> => {
   } catch {
     await TrackPlayer.setupPlayer();
     await TrackPlayer.updateOptions({
+      androidOptions: {
+        stoppingAppRemovesNotification: true
```

Comment:
If this is true then `stoppingAppPausesPlayback` must also be true. It's explained in the docs.

Reply:
@mpivchev should we default this to true?

Reply:
It seems to me this is the behavior most people will want.

Reply:
@mpivchev @dcvz what about making an enum like StopMode which has PausePlayback PausePlaybackAndRenoveNotificstion

Then we deprecate the old stoppingAppPausesPlayback property.

Reply:
@jspizziri Good idea :) Will do that.